ci-operator/step-registry/openshift-qe/max-namespaces/OWNERS

```diff
@@ -0,0 +1 @@
+../OWNERS
```

Comment:
Sahil, add your github user name to the owners file at the base of the openshift-qe folder under step-registry. maybe since you're not listed it didnt run your test/comment

Reply:
Yup Thanks